### PR TITLE
[Metricbeat] Use timestamp from CloudWatch for events

### PR DESCRIPTION
**Pull request by kaiyan-sheng**
This PR is to use timestamp from CloudWatch ListMetric API instead of the current timestamp to construct events.

## Checklist

- [x] My code follows the style guidelines of this project
- [ ]~~I have commented my code, particularly in hard-to-understand areas~~
- [ ]~~I have made corresponding changes to the documentation~~
- [ ]~~I have made corresponding change to the default configuration files~~
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

### CODE REVIEW

File: x-pack/metricbeat/module/aws/aws.go [hunk 1]
```diff
@@ -208,6 +208,8 @@ func InitEvent(regionName string, accountName string, accountID string) mb.Event
 	if accountID != "" {
 		event.RootFields.Put("cloud.account.id", accountID)
 	}
+
+	event.Timestamp = timestamp
```
Comment (ChrsMark) [inline x-pack/metricbeat/module/aws/aws.go:212]: I 've seen in the past that this kind of assignment was not working, could you confirm? 
Nevertheless I would go with sth like:
```
event := mb.Event{
  Timestamp: timestamp,
  RootFields: common.MapStr{},
```
wdyt?

**Review by ChrsMark — COMMENTED** [review]
Looks good to me! Wondering if it would worth adding unit-tests for this change? Was really helpful in Prometheus module. Not sure if possible here though.

File: x-pack/metricbeat/module/aws/aws.go (hunk 1 above)
Comment (kaiyan-sheng) [reply x-pack/metricbeat/module/aws/aws.go]: Good idea! Done!

**Comment by kaiyan-sheng** [comment]
> Looks good to me! Wondering if it would worth adding unit-tests for this change? Was really helpful in Prometheus module. Not sure if possible here though.

Thanks for the review @ChrsMark ! I just added a unit test for this! Definitely good to have a test in there even with mocked apis.

**Review by ChrsMark — APPROVED** [review]

**Pull request merged**